### PR TITLE
[node-manager] discover_packages_proxy_addresses hook fix

### PR DESCRIPTION
**Pull request by borg-z**
Signed-off-by: borg-z <me@nikolay-z.top>

## Description

Fixed the behavior of the discover_packages_proxy_addresses hook that caused a static node to be created with an empty PACKAGES_PROXY_ADDRESSES variable, resulting in a bootstrap error.

## Why do we need it, and what problem does it solve?

This change resolves an issue where nodes would fail to bootstrap due to missing package proxy addresses

## What is the expected result?

After applying these changes, nodes should bootstrap successfully without errors related to missing package proxy addresses.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager, registry-packages-proxy
type: fix
summary: Fixed issue causing static nodes to be created with empty package proxy addresses, leading to bootstrap errors.
impact: Nodes will now bootstrap successfully without errors related to missing package proxy addresses.)
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/vcd'...

File: modules/040-node-manager/hooks/discover_packages_proxy_addresses.go [hunk 1]
```diff
@@ -34,7 +34,8 @@ const (
 )
 
 var _ = sdk.RegisterFunc(&go_hook.HookConfig{
-	Queue: "/modules/node-manager",
+	OnBeforeHelm: &go_hook.OrderedConfig{Order: 1},
```
Comment (RomanenkoDenys) [inline modules/040-node-manager/hooks/discover_packages_proxy_addresses.go:37]: in general we use 1 in the node-manager module for migrations, so 5 will be the reliable value here

**Review by RomanenkoDenys — COMMENTED** [review]
some nit

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Review by name212 — APPROVED** [review]

**Pull request merged**